src/imgui/ImGuiMachine.cc

```diff
@@ -279,6 +267,99 @@ void ImGuiMachine::showMenu(MSXMotherBoard* motherBoard)
 			});
 		}
 
+		setupSettingsOpen = im::Menu("Setup settings", true, [&]{
+			ImGui::TextUnformatted("When openMSX starts:");
+			ImGui::RadioButton("Load a machine", &startType, StartType::MACHINE);
+			HelpMarker("Select this option to load the specified machine when openMSX starts up, instead of a setup.");
+			im::Indent([&] {
+				im::Disabled(startType != StartType::MACHINE, [&] {
```

Comment:
Because this combination of `im::Indent()`  and `im::Disabled()` occurs often (and it's annoying that it indents the c++ code twice), we've created a utility `im::DisabledIndent()`.